fair/static/js/lunch_tickets_check_in/TicketList/index.js

```diff
@@ -81,8 +81,16 @@ class TicketList extends Component {
       API.search(searchQuery)
           .then(response => {
             const sortedTickets = sortBy(response.data.result, 'date');
+            let finalTickets = []
```

Comment:
Regarding this I believe that it is better to filter on the fair already wh ndoing the API-call to the AIS-server. This API is defined in the file fair\api.py and the function which returns the search query is the one named lunchtickets_search. It looks like it already filters on the current fair but there is some bug. Therefore I think it could be useful to start by investigating this bug. Perhaps it is just some syntax error in this API function. :) No changes will then be needed in the javascript code.